bugbug/generative_model_tool.py

```diff
@@ -7,19 +7,28 @@
 from typing import Any
 
 from langchain_community.llms import HumanInputLLM
-from langchain_openai import ChatOpenAI
+from langchain_openai import AzureChatOpenAI, ChatOpenAI
 
 from bugbug.utils import get_secret
 
 
 def create_llm(llm):
+    temperature = 0.2
```

Comment:
```suggestion
    openai_temperature = 0.2
```

Reply:
fixed

Reply:
It seems that you added some unrelated changes, which were not there when you opened the PR. I guess you copied commits from the master branch.